lib/mongoid/association/nested/nested_buildable.rb

```diff
@@ -56,7 +56,7 @@ def update_only?
         # @param [ Class ] klass The class we're trying to convert for.
         # @param [ String ] id The id, usually coming from the form.
         #
-        # @return [ BSON::ObjectId, String, Object ] The converted id.
+        # @return [ BSON::ObjectId | String | Object ] The converted id.
```

Comment:
Shouldn't this just be Object?

Reply:
I'm okay with leaving this as it is if you prefer

Reply:
It could be Object (as obviously Object is a superset of the others) but I think its reasonable to provide color on the "usually" expected type(s) here. So would suggest to leave this as I have it.